docs/application/native/tutorials/details/app-filtering.md

```diff
@@ -186,6 +186,7 @@ package:
 | `http://tizen.org/feature/minicontrol`   | Specify this key, if the application requires the minicontrol feature. | 4.0   |
 | `http://tizen.org/feature/multi_point_touch.pinch_zoom` | Specify this key, if the application requires a pinch-zoom gesture feature. | 2.2.1 |
 | `http://tizen.org/feature/multi_point_touch.point_count` | Specify this key with a specific number (`int` type), if the application requires the minimum of specified number of simultaneous touches in a multi-point touch. | 2.2.1 |
+| `http://tizen.org/feature/multimedia.media_codec`     | Specify this key, if the application requires the media codec feature. | 4.0   |
```

Comment:
Question. We have feature tables for mobile and wearable. This feature is added to mobile feature table only. Doesn't the wearable support this feature?

Reply:
It is already added in wearable section. I think the engineer who was in charge of modified the wearable section only at that time.

Reply:
I confirmed this change.